spec/rubocop/cop/rspec/single_argument_message_chain_spec.rb

```diff
@@ -47,6 +47,49 @@
         end
       RUBY
     end
+
+    context 'with single-key hash argument' do
+      it 'reports an offence' do
+        expect_violation(<<-RUBY)
+          before do
+            allow(foo).to receive_message_chain(bar: 42)
+                          ^^^^^^^^^^^^^^^^^^^^^ Use `receive` instead of calling `receive_message_chain` with a single argument.
+          end
+        RUBY
+      end
+
+      include_examples(
+        'autocorrect',
+        'before { allow(foo).to receive_message_chain(bar: 42) }',
+        'before { allow(foo).to receive(:bar) { 42 } }'
+      )
+
+      include_examples(
+        'autocorrect',
+        'before { allow(foo).to receive_message_chain("foo bar" => 42) }',
+        'before { allow(foo).to receive("foo bar") { 42 } }'
```

Comment:
lol, I was so concerned to handle all hash cases that I forgot they are used in a `receive` call. Guess all this safety is unnecessary